indico/modules/events/templates/display/administrative/event.html

```diff
@@ -0,0 +1,89 @@
+{% from 'events/display/common/_legacy.html' import render_event_time, render_location %}
+{% from 'events/display/common/_manage_button.html' import render_manage_button %}
+
+<table class="eventWrapper">
+    <tr>
+        <td>
+            <div class="manage-button-container">
+                {{ render_manage_button(event, 'EVENT', toggle_notes=false, show_note_operations=true) }}
+            </div>
+            <div class="reportNumberHeader">
+                {% for ref in event.references %}
+                    {{ ref.value }}
+                    {% if not loop.last %}
+                        <br>
+                    {% endif %}
+                {% endfor %}
+            </div>
+            {% block header %}
+                <br>&nbsp;<br>
+                <div class="eventHeader">
+                    {{ theme_settings.title_fr or
+                       'ORGANISATION EUROP&Eacute;ENNE POUR LA RECHERCHE NUCL&Eacute;AIRE' | safe }}
```

Comment:
I'm usually wary of using unicode chars in source code. It's true that Jinja can handle it fine, but it's a matter of keeping it ASCII-friendly.